### PR TITLE
Add /postal cheat

### PR DESCRIPTION
**Pull request by Wanted9811**

### CODE REVIEW

File: src/handlers/feature_handlers.cpp [hunk 1]
```diff
@@ -1997,105 +2046,8 @@ void BridgeFerryHandler::Cull(int lod) {
 }
 
 void BridgeFerryHandler::Draw(int lod) {
-    auto bangerInst = reinterpret_cast<dgUnhitYBangerInstance*>(this);
```
Comment (0x1F9F1) [inline src/handlers/feature_handlers.cpp:2000]: Why was this removed? Won't it stop bridges from being drawn?

Comment (Wanted9811) [reply src/handlers/feature_handlers.cpp:2000]: Nah it won't stop them, I just called `dgBangerInstance::Draw` instead of what I rewrote inside the function, I will rewrite it latter more nicely because it had glitches so that's why I removed the rewritten one.

Comment (0x1F9F1) [reply src/handlers/feature_handlers.cpp:2000]: Ah I see. it's the "call original" which confused me, because the original Draw function does nothing. It's Cull which originally called it.

**Review by 0x1F9F1 — APPROVED** [review]
